gui/wxpython/startup/utils.py

```diff
@@ -18,41 +18,48 @@
 
 import os
 import shutil
+import tempfile
+import sys
 
 
 def get_possible_database_path():
-    """Finds a path to what is possibly a GRASS Database. 
+    """Finds the directory to what is possibly a GRASS Database.
 
     Looks for directory named grassdata in the usual locations.
 
-    Returns the path as a string or None if nothing was found, so the
-    return value can be used to test if the directory was found.
+    Returns the path as a string or None if nothing was found, so the return 
+    value can be used to test if the directory was found.  
     """
-    home = os.path.expanduser('~')
+    home = os.path.expanduser('~')           
+    path = None     
     # try some common directories for grassdata
     # grassdata (lowercase) in home for Linux (first choice)
     # Documents and My Documents for Windows
     # potential translations (old Windows and some Linux)
     # but ~ and ~/Documents should cover most of the cases
-    # ordered by preference and then likelihood
+    # ordered by platform preference      
+            
+    # Search independent "grassdata" directories      
     candidates = [
-        os.path.join(home, "grassdata"),
-        os.path.join(home, "Documents", "grassdata"),
-        os.path.join(home, "My Documents", "grassdata"),
+        os.path.join(home),
+        os.path.join(home, "Documents")
```

Comment:
The point of trying "My Documents" is that we are not following the real API here of Windows or any other platform, so we are just adding this to the list of guesses. I put "My Documents" there because based on what I found at that time it seemed that it is a possible option (that there will be no "Documents" but only "My Documents"). This is does not seem to be the case for most recent Windows, but might be for some which are in use. I simply don't know, so I would prefer to keep "My Documents" there now, although I would like to see some analysis of this and a better solution in the (possibly distant) future.